libraries/plugins/elasticsearch/elasticsearch_plugin.cpp

```diff
@@ -488,4 +516,124 @@ void elasticsearch_plugin::plugin_startup()
    ilog("elasticsearch ACCOUNT HISTORY: plugin_startup() begin");
 }
 
+operation_history_object elasticsearch_plugin::get_operation_by_id(operation_history_id_type id)
+{
+   const string operation_id_string = std::string(object_id_type(id));
+
+   const string query = R"(
+   {
+      "query": {
+         "match":
+         {
+            "account_history.operation_id": )" + operation_id_string + R"("
```

Comment:
A `"` is missing before `)" + operation_id_string`.

It seems this code is not used anywhere though.

@sschiessl-bcp I found that you were in the conversation, do you remember anything about this?